src/main.py

```diff
@@ -1,9 +1,12 @@
+
+import csv
 import os
 
+from pathlib import Path
 from fastapi import FastAPI, Request, Response, status
 
 from constants import JSONLD
-from store import StubStore
+from store import StubStore, StubMetadataStore
```

Comment:
slight naming mixup.

the origin `StubStore` needs to be renamed to `StubMetadataStore` (and update anywhere it's referenced.

your new store needs to be `StubCsvStore`.